src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -332,6 +332,7 @@ class EnvConfigurationWithoutRootNode implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
+        throw new \Exception();
```

Comment:
this test from #27472 is actually invalidated due #27710, we should pass some config to ensure the treebuilder is called

Reply:
up for a PR?